superflore/generators/ebuild/ebuild.py

```diff
@@ -250,9 +244,10 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += "\"\n\n"
 
         # SLOT
-        ret += "SLOT=\"{}\"\n".format(self.distro)
+        ret += "SLOT=\"0\"\n"
         # CMAKE_BUILD_TYPE
-        ret += "CMAKE_BUILD_TYPE=RelWithDebInfo\n"
+        if self.name == "catkin":
+            ret += "BUILD_BINARY=\"0\"\n"
```

Comment:
@tfoote Yes, I set this up as an eclass variable which, if present, will set the `-DCATKIN_BUILD_BINARY_PACKAGE` flag to its value. Otherwise, that flag defaults to `1` (this just simplifies the catkin ebuild, making it more like the others).